cowprotocol/cowswap
# chore(tech-debt): manually split large libs into their own chunks

# Summary

Manually split chunks into smaller chunks

Comparison:

||Before|After|
|-|-|-|
|Graph view| ![image](https://github.com/cowprotocol/cowswap/assets/43217/4530fe99-2614-4f54-a51e-ed8f1f38d2fa) | ![Screenshot 2023-08-31 at 17 07 44](https://github.com/cowprotocol/cowswap/assets/43217/c2037be5-e6e9-4ec4-b09c-0bc2685954ac) |
|Chunks| ![image](https://github.com/cowprotocol/cowswap/assets/43217/3f5127ac-463e-4dad-8f27-118f18396adc) | ![image](https://github.com/cowprotocol/cowswap/assets/43217/f903c603-cb71-47cf-9926-aa7e27442cfc) |

This is an attempt to fix the Vercel build which silently fails on https://github.com/cowprotocol/cowswap/pull/3067

Now there are more chunks, and the biggest chunk is 1.7mB/585kB vs 6.7mB/1.8mB before.

## ⚠️ WARNING ⚠️ 

With the further chunk split, the app doesn't load all at once. It might be perceived as slower to load.
Let's verify this is not causing a too big UX impact

I also removed the Sentry plugin as it was there only for uploading the generated sourcemaps. But the sourcemaps generation have been disable as that also caused the vercel build to fail...

# To Test

1. Open the app in a private browsing window
* Observe it loading

## Load comparison

### Before

https://github.com/cowprotocol/cowswap/assets/43217/164d6ecb-f412-4d95-b231-57f3ba79c56d

### Now

https://github.com/cowprotocol/cowswap/assets/43217/d88070ab-2790-4890-b017-ced3f000cd31
